ppa/archive/templates/admin/archive/digitizedwork/bulk_add_collections.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'admin/base_site.html'  %}
+{% load i18n grp_tags %}
+{# Adapts Grappelli form styles for use in an intermediate view #}
+{% block title %} Bulk Add to Collections | {% get_site_title %} {% endblock %}
+{% block breadcrumbs %}
+    {% if not is_popup %}
+        <ul class="grp-horizontal-list">
+            <li><a href="{% url 'admin:index' %}">{% trans "Home" %}</a></li>
+            <li><a href="{% url 'admin:app_list' app_label='archive' %}">Archive</a></li>
+            <li><a href="{% url 'admin:archive_digitizedwork_changelist'%}">Digitized Works</a></li>
+            <li>Bulk Add to Collections</li>
+        </ul>
+    {% endif %}
+{% endblock %}
+
+{% block content_title %}
+    <h1>Bulk Add to Collections</h1>
```

Comment:
After using this a couple times, I think "bulk" should be dropped here.  It's doesn't add anything, and is potentially confusing.  Maybe relabel to be more explicit: "Add digitized works to collections"?

I think it's fine to keep the "bulk" terminology internally within the code, but suggest dropping it anywhere else that is public facing or API-ish (like the url names).

Reply:
I've revised this for clarity/public-facing terminology.